src/Features/CSharp/Portable/IntroduceVariable/CSharpIntroduceVariableService_IntroduceLocal.cs

```diff
@@ -47,14 +50,25 @@ protected override Document IntroduceLocal(
             ? TokenList(ConstKeyword)
             : default;
 
+        var updatedExpression = expression.WithoutTrivia();
+        var csOptions = (CSharpSimplifierOptions)options.SimplifierOptions;
+
+        // If the target-type new syntax is preferred and "var" is not preferred under any circumstance, then we use the target-type new syntax.
+        // The approach is not exhaustive. We aim to support codebases that rely strictly on the target-type new syntax (i.e., no "var").
+        if (csOptions.ImplicitObjectCreationWhenTypeIsApparent.Value && csOptions.GetUseVarPreference() == UseVarPreference.None
+            && expression is ObjectCreationExpressionSyntax objectCreationExpression)
+        {
+            updatedExpression = ImplicitObjectCreationExpression(objectCreationExpression.ArgumentList, objectCreationExpression.Initializer);
```

Comment:
you removed the trivia from expression when making updatedExpression at the top of the method.  but not here.  consider either doign the same here, or doign `updatedExpression is ObjectCreationExpressionSyntax ...` to ensure that original op wasn't lost.